modules/dedicated-enable-private-cluster.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * cloud_infrastructure_access/dedicated-private-cluster.adoc
+
+[id="dedicated-enable-private-cluster-new"]
+= Enabling Private Cluster on a new cluster
+
+.Prerequisites
+
+* AWS VPC Peering, VPN, DirectConnect, or TransitGateway has been previously configured to allow private access.
+
+.Procedure
+
+. In the OpenShift Cluster Manager, click *Create cluster* and select *{product-title}*.
+. Configure your Cluster details, then select *Advanced* in the Networking section.
+. Determine your CIDR requirements for your network.
```

Comment:
Is this enough information for the user to correctly configure their CIDR range? And how do they set them in this flow?